schemas/descriptors/display/alternateDisplayInfo.schema.json

```diff
@@ -106,6 +106,9 @@
         },
         {
           "required": ["meta:enum"]
+        },
+        {
+          "required": ["xdm:excludeMetaEnum"]
```

Comment:
@bisin-adobe , Why do we need to make this required when the same descriptor is used for other purposes as well ?

Reply:
@mjasrotia 
That is an anyOf attribute. It means one of the required field should be available in the payload.
If you want to exclude meta:enum, "xdm:excludeMetaEnum" is mandatory and this json will do the validation